platforms/ios/build_framework.py

```diff
@@ -406,11 +406,11 @@ def makeFramework(self, outdir, builddirs):
             for dirname, dirs, files in os.walk(os.path.join(dstdir, "Headers")):
                 for filename in files:
                     filepath = os.path.join(dirname, filename)
-                    with open(filepath) as file:
+                    with codecs.open(filepath,"r","utf-8") as file:
```

Comment:
In [`gen_objc.py`](https://github.com/opencv/opencv/blob/4.5.4/modules/objc/generator/gen_objc.py#L1432) we using:

```
with io.open(filepath, encoding="utf-8", errors="ignore") as file:
```

---

Please use space after comma.

Reply:
codecs.open() is used elsewhere in this file, so I chose it for consistency. Spaces have been added.